app/views/pulitzer/post_type_versions/_index.html.erb

```diff
@@ -2,9 +2,13 @@
 <h3>"<%= @post_type.name %>" Template Versions</h3>
 <div class="pulitzer-row margin-bottom list-item">
   <% if Pulitzer.skip_metadata_auth? || self.instance_eval(&Pulitzer.metadata_closure) %>
-    <%= ajax_link "create a new version of this template",
+    <%= ajax_link "Create a blank version of this template",
           post_type_versions_path(post_type_version: {post_type_id: @post_type.id}),
           {}, '#post_type_versions_container', http_method: 'POST', insert_method: 'append' %>
+  <br>
+  <%= ajax_link "Clone the published version of this template",
```

Comment:
@gomayonqui I think we need a conditional here to only show this if there is already a published version of the template.